components/brave_wallet_ui/components/shared/create-network-icon/index.tsx

```diff
@@ -31,7 +31,7 @@ function CreateNetworkIcon (props: Props) {
   )
 
   const networkImageURL = stripERC20TokenImageURL(network?.iconUrls[0])
-  const isRemoteURL = isRemoteImageURL(networkImageURL)
+  const isRemoteURL = isFromDifferentOrigin(networkImageURL)
```

Comment:
this is confusing since `isFromDifferentOrigin` no longer means it's a remote URL.